zerver/openapi/zulip.yaml

```diff
@@ -10044,6 +10044,13 @@ paths:
           schema:
             type: boolean
           example: true
+      x-response-description: |
+        #### Return values
```

Comment:
I think we don't want this heading along, since it's a duplicate.

Reply:
This already got merged with the changed description :)